lua/complementree/sources.lua

```diff
@@ -162,6 +162,58 @@ local function lsp_completedone(completed_item)
   end
 end
 
+M.filepath = function(_, _, _, _)
+  local MAX_FILEPATH_DEPTH = 8 -- GET RID OF THESE
+  local MAKE_RELATIVE = true
```

Comment:
Maybe turn the `filepath` function into a function that returns a source, and use the parameters of the outer function to configure the inner function ?